packages/bananagram-common/src/types.ts

```diff
@@ -0,0 +1,120 @@
+export interface TokenData<T> {
+  id: number;
+  token: T;
+}
+
+type BananagramTokens =
+  | 'a'
+  | 'b'
+  | 'c'
+  | 'd'
+  | 'e'
+  | 'f'
+  | 'g'
+  | 'h'
+  | 'i'
+  | 'j'
+  | 'k'
+  | 'l'
+  | 'm'
+  | 'n'
+  | 'o'
+  | 'p'
+  | 'q'
+  | 'r'
+  | 's'
+  | 't'
+  | 'u'
+  | 'v'
+  | 'w'
+  | 'x'
+  | 'y'
+  | 'z';
+type WildBananagramTokens = BananagramTokens | '*';
+
+export const numberOfBananagramTokens: Record<BananagramTokens, number> = {
```

Comment:
Could  we extract this `Record<BananagramTokens, number>` into a common type called `NumberOfTokens`.
e.g:
```ts
const type NumberOfTokens<T extends string> = Record<T, number>
```
and then use it in all of the places we've used `Record<Something, number>.

Reply:
So it would look like
```ts
export const numberOfBanangramTokens: NumberOfTokens<BanangramTokens> = {
  ...
}
```

Reply:
added type for Record<T, number> 30b773c